content/docs/gto/get-started-dvc.md

```diff
@@ -0,0 +1,115 @@
+# Get Started DVC
+
+To leverage concepts of Model and Data Registries in a more explicit way, you
+can denote the `type` of each output. This will let you browse models and data
+separately, address them by `name` in `dvc get`, and eventually, see them in DVC
+Studio.
+
+Let's start with marking an artifact as data or model.
+
+If you're using `dvc add` to track your artifact, you'll need to run:
+
+```dvc
+$ dvc add mymodel.pkl --type model
+```
+
+If you're producing your models in DVC pipeline, you'll need to add
+`type: model` to `dvc.yaml` instead:
+
```

Comment:
Now I think there is a user case this change may not support that well. One of our prospects asked to allow a single file (let's say `mymodel.pkl`) to be referenced as several GTO models (e.g. `model1` and `model2` - these are `name`s). Since moving to DVC makes `path` essential (instead of `name`), I don't see how that feature would fit here. 🤔 

The motivation is to be able to promote `model1` and `model2` to different stages at different moments of time separately. To clarify, let's assume there are two populations `mymodel.pkl` should be applied for. You can create stages like `populationA-prod`, `populationA-staging` and `populationB-prod`, `populationB-staging`, if you have many populations, this would make things cumbersome. The solution was to introduce `model1` (for `populationA`) and `model2` (for `populationB`). That required this feature.

The only workaround I see now is to create a "mirror file" with `cp mymodel.pkl mymodel-for-populationB.pkl` in some DVC PL stage. Or keep this `name`:`path` mapping outside of DVC somehow. Do you see any other solutions guys? WDYT?

Reply:
[related](https://github.com/iterative/mlem.ai/pull/323#discussion_r1131390110)

Reply:
Agree with @omesser that it's related the discussion above. Take a look at the [top-level plots schema](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#plots), where plots may be identified by either path or an arbitrary name. Feels like following a similar syntax may be best here.